playwright/integrations/unauthenticated-chat-with-ooh.spec.ts

```diff
@@ -0,0 +1,40 @@
+import fetchOmnichannelConfig from '../utils/fetchOmnichannelConfig';
+import fetchTestPageUrl from '../utils/fetchTestPageUrl';
+import { test, expect } from '@playwright/test';
+
+const testPage = fetchTestPageUrl();
+const omnichannelConfig = fetchOmnichannelConfig('UnauthenticatedChatWithOOOH');
```

Comment:
Can we rename it to `UnauthenticatedChatOutsideOfOperatingHours`?

It'd be easier for new contributors to understand the naming.